### ISSUE TITLE
IDE Fatal Errors

### ISSUE THREAD

**Issue by futdev** [hsz/idea-nodesecurity#4]
This is upon starting PhpStorm after installing the plugin.

```
null
Unexpected character 'EOI(null)' at position '3', expecting '[DOT]'
	at com.github.zafarkhaja.semver.VersionParser.consumeNextCharacter(VersionParser.java:516)
	at com.github.zafarkhaja.semver.VersionParser.parseVersionCore(VersionParser.java:288)
	at com.github.zafarkhaja.semver.VersionParser.parseValidSemVer(VersionParser.java:255)
	at com.github.zafarkhaja.semver.VersionParser.parseValidSemVer(VersionParser.java:195)
	at com.github.zafarkhaja.semver.Version.valueOf(Version.java:265)
	at mobi.hsz.idea.nodesecurity.models.Advisory.isVulnerable(Advisory.kt:36)
	at mobi.hsz.idea.nodesecurity.utils.VulnerabilitiesScanner$Companion$scan$1.doResume(VulnerabilitiesScanner.kt:29)
	at kotlin.coroutines.experimental.jvm.internal.CoroutineImpl.resume(CoroutineImpl.kt:54)
	at kotlin.coroutines.experimental.SequenceBuilderIterator.hasNext(SequenceBuilder.kt:129)
	at mobi.hsz.idea.nodesecurity.components.NodeSecurityProjectComponent$projectOpened$1.run(NodeSecurityProjectComponent.kt:25)
	at com.intellij.openapi.project.DumbServiceImpl.b(DumbServiceImpl.java:170)
	at com.intellij.ide.startup.impl.StartupManagerImpl.a(StartupManagerImpl.java:367)
	at com.intellij.ide.startup.impl.StartupManagerImpl.b(StartupManagerImpl.java:182)
	at com.intellij.openapi.project.DumbServiceImpl.b(DumbServiceImpl.java:170)
	at com.intellij.ide.startup.impl.StartupManagerImpl.a(StartupManagerImpl.java:396)
	at com.intellij.ui.GuiUtils.invokeLaterIfNeeded(GuiUtils.java:377)
	at com.intellij.ide.startup.impl.StartupManagerImpl.runWhenProjectIsInitialized(StartupManagerImpl.java:398)
	at com.intellij.openapi.project.DumbServiceImpl.runWhenSmart(DumbServiceImpl.java:162)
	at com.intellij.ide.startup.impl.StartupManagerImpl.c(StartupManagerImpl.java:182)
	at com.intellij.ide.startup.impl.StartupManagerImpl.access$400(StartupManagerImpl.java:58)
	at com.intellij.ide.startup.impl.StartupManagerImpl$1.a(StartupManagerImpl.java:212)
	at java.util.ArrayList.forEach(ArrayList.java:1251)
	at com.intellij.ide.startup.impl.StartupManagerImpl$1.run(StartupManagerImpl.java:212)
	at com.intellij.openapi.project.DumbServiceImpl.b(DumbServiceImpl.java:170)
	at com.intellij.ide.startup.impl.StartupManagerImpl.a(StartupManagerImpl.java:396)
	at com.intellij.ui.GuiUtils.invokeLaterIfNeeded(GuiUtils.java:377)
	at com.intellij.ide.startup.impl.StartupManagerImpl.runWhenProjectIsInitialized(StartupManagerImpl.java:398)
	at com.intellij.openapi.project.DumbServiceImpl.runWhenSmart(DumbServiceImpl.java:162)
	at com.intellij.ide.startup.impl.StartupManagerImpl.runPostStartupActivities(StartupManagerImpl.java:200)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.a(ProjectManagerImpl.java:415)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:315)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.a(LaterInvocator.java:424)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:407)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:762)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:732)
	at com.intellij.ide.IdeEventQueue.g(IdeEventQueue.java:822)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:650)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:366)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by hsz** [comment]
@InkedCurtis  Thank you for the report (and sorry for delay!). Issue is fixed with 662406d and will be released with v0.3.

**Comment by hsz** [comment]
Released with v0.3.0
